src/cljc/quil/snippets/output.cljc

```diff
@@ -18,14 +18,12 @@
 
 (defsnippet save
   "save"
-  {:renderer :p3d}
+  {:renderer :p3d
+   :settings q/no-loop}
 
   (q/camera 150 150 150 0 0 0 0 0 1)
   (q/box 100)
   (q/save "generated/box.png")
-  (comment "stop sketch after saving image")
-  (comment "otherwise it will show save dialog")
-  (comment "on every iteration")
   (q/exit))
```

Comment:
Wny doesn't q/exit help here? It supposed to solve the same problem as q/no-loop.

Reply:
Oh right, how did I miss that? Yes `q/exit` calls p5js `remove` which works here: https://editor.p5js.org/anthonygalea/sketches/pnmR05vm4

Reply:
Oh, `q/exit` is not there when the sketch is opened in the editor:
<img width="680" alt="Screenshot" src="https://user-images.githubusercontent.com/1301852/55689356-b2640d00-5983-11e9-823b-49fdb334a03e.png">

Reply:
Ah ok got it. The call to `exit` was simply added after snippets data generation for quil site. Will remove the commit containing the `no-loop` in this pull request and submit another pull request to update snippets data in quil-site.